src/main/scala/net/snowflake/spark/snowflake/SnowflakeTelemetry.scala

```diff
@@ -550,3 +579,25 @@ object TelemetryOOBTags {
   val CTX_PROTOCAL = "ctx_protocol"
   val CTX_USER = "ctx_user"
 }
+
+private[snowflake] trait TelemetryMessageSender {
+  def send(telemetry: Telemetry, logs: List[(ObjectNode, Long)]): Unit
+}
+
+private final class RealTelemetryMessageSender extends TelemetryMessageSender {
```

Comment:
I don't like this name.

Reply:
any other idea? What about `SnowflakeTelemetryMessageSender` ?

Reply:
much better